### PR TITLE
Update macOS doc to 3.0.0

### PR DESCRIPTION
**Pull request by jensflorian**
Signed-off-by: Jens Schittenhelm <github@jensflorian.com>

### CODE REVIEW

File: installation/macos.md [hunk 1]
```diff
@@ -21,20 +21,21 @@ If you're unfamiliar with using the macOS terminal, then feel free to follow the
 ## Installation
 
 openHAB is packaged as a platform independent `zip` archive. Installation is as simple as unziping it into a unique folder.
-First, make sure that your system meets the [prerequisites](index.html#prerequisites).
+First, make sure that your system meets the [prerequisites](index.html#prerequisites). When you are running macOS BigSur or higher, make sure
```
Comment (Confectrician) [inline installation/macos.md:24]: One line per sentence please.

```suggestion
First, make sure that your system meets the [prerequisites](index.html#prerequisites).
When you are running macOS BigSur or higher, make sure
```

File: installation/macos.md [hunk 2]
```diff
@@ -73,28 +74,26 @@ From inside openHAB's root directory (e.g. `~/openhab/), run openHAB's startup s
...
+To exit, use '<ctrl-d>' or 'logout'.
+
 
 openhab>
 ```
 
-Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/beginner/1sttimesetup.html).
-
-![](images/Accueil_Openhab_2.png)
+Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/beginner/1sttimesetup.html). If you installed openHAB on a different device, replace localhost with the IP address of the device.
```
Comment (Confectrician) [inline installation/macos.md:96]:
```suggestion
Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/beginner/1sttimesetup.html).
If you installed openHAB on a different device, replace localhost with the IP address of the device.
```

File: installation/macos.md [hunk 3]
```diff
@@ -111,17 +110,17 @@ cd ~/openhab
 sudo runtime/bin/update
 ```
 
-You can also specify any version as a parameter. For example, to switch back to 2.0.0:
+You can also specify any version as a parameter. For example, to switch back to 2.5.11:
```
Comment (Confectrician) [inline installation/macos.md:113]:
```suggestion
You can also specify any version as a parameter.
For example, to switch back to 2.5.11:
```

**Review by Confectrician — COMMENTED** [review]
Thanks and congrats to your first PR here. 🙂

Looks good in general.
I have left some minor comments.

We are doing only one sentence per line for a better version tracking.
The website build will take care of a propper paragraph presentation on the website. :)

File: installation/macos.md [hunk 4]
```diff
@@ -73,28 +74,27 @@ From inside openHAB's root directory (e.g. `~/openhab/), run openHAB's startup s
...
+To exit, use '<ctrl-d>' or 'logout'.
+
 
 openhab>
 ```
 
-Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/beginner/1sttimesetup.html).
-
-![](images/Accueil_Openhab_2.png)
+Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorials/first_steps.html).
```
Comment (jensflorian) [inline installation/macos.md:96]: I also changed the link here, please check, that this is now correct.

Comment (jensflorian) [inline installation/macos.md:96]: I changed the link to first_steps.html, because in preview I got a 404

Comment (Confectrician) [reply installation/macos.md:97]:
```suggestion
Without closing the terminal, open your favorite web browser and type the following URL: [http://localhost:8080](http://localhost:8080), you should see the openHAB welcome screen, and you're all set to [using openHAB]({{base}}/tutorial/first_steps.html). 
```

on letter too much. :)

Comment (jensflorian) [reply installation/macos.md:97]: Ah, the folder changed too. Thanks

**Review by Confectrician — APPROVED** [review]
LGTM Thanks. 👍

**Pull request merged**